### ISSUE TITLE
bug: AndroidManifest acitivity didn't work for my app

### ISSUE THREAD

**Issue by aces-tm** [ionic-team/capacitor#4859]
## Bug Report

I updated my app to Capacitor 3 and used AndroidManifest.xml that is provided with a fresh build of Ionic app. When I added my store the PlayStore it was approved and I could download it, but I couldn't launch the app. The app was not listed among the lists of the the apps at all, but I could find it in settings without option to launch. 

I looked into the difference of AndroidManifest.xml that was used in Cap 2 and added missing `intent-filter` and the issue was fixed. 

I am not sure if this is an issue with my app or with something else, but since I lost 3 days on trying to figure it out, I thought it'd be good to report it. 

```

            <intent-filter>
                <action android:name="android.intent.action.VIEW" />

                <category android:name="android.intent.category.DEFAULT" />
                <category android:name="android.intent.category.BROWSABLE" />

                <data android:scheme="@string/custom_url_scheme" />
            </intent-filter>
            <intent-filter android:autoVerify="true">
                <action android:name="android.intent.action.VIEW" />

                <category android:name="android.intent.category.DEFAULT" />
                <category android:name="android.intent.category.BROWSABLE" />

                <data
                    android:scheme="https"
                    android:host="ioncourt.com" />
            </intent-filter>
```

### Capacitor Version

```
Latest Dependencies:

  @capacitor/cli: 3.1.2
  @capacitor/core: 3.1.2
  @capacitor/android: 3.1.2
  @capacitor/ios: 3.1.2

Installed Dependencies:

  @capacitor/cli: 3.1.1
  @capacitor/android: 3.1.1
  @capacitor/core: 3.1.1
  @capacitor/ios: 3.1.1

[success] iOS looking great! 👌
[success] Android looking great! 👌
```

### Other Technical Details

`npm --version` output:

`node --version` output:

`pod --version` output (iOS issues only):

**Comment by elylucas** [comment]
Hi @aces-tm 

What did your intents look like before you added the ones from cap 2? By default, a new cap 3 app comes with the following intents that allow it to be installed and available via the app launcher:

```
<intent-filter>
                <action android:name="android.intent.action.MAIN" />
                <category android:name="android.intent.category.LAUNCHER" />
            </intent-filter>
```

**Comment by aces-tm** [comment]
@elylucas It looked like that. 

I built a brand new app and copied over AndroidManifest.xml. Then I used git tool in VS Code to compare differences and there were none after I replaced app name. I also completely removed android platform and added new. Finally I went to the commit before I updated to cap3 and just copied AndroidManifest and it started to work. The difference in the code were intent filters posted above.

I am still very puzzled. I fixed my issue, but it should have worked without additional intent filters.
